apps/etterlatte-behandling/src/main/kotlin/behandling/BehandlingService.kt

```diff
@@ -830,13 +833,28 @@ internal class BehandlingServiceImpl(
             .also {
                 behandlingDao.lagreViderefoertOpphoer(behandlingId, viderefoertOpphoer)
                 behandlingDao.lagreStatus(it)
+                if (behandling.sak.sakType == SakType.OMSTILLINGSSTOENAD) {
+                    beregningKlient.slettAvkorting(behandling.id, brukerTokenInfo)
+                }
             }
     }
 
-    override fun fjernViderefoertOpphoer(
+    override suspend fun fjernViderefoertOpphoer(
         behandlingId: UUID,
-        kilde: Grunnlagsopplysning.Kilde,
-    ) = behandlingDao.fjernViderefoertOpphoer(behandlingId, kilde)
+        brukerTokenInfo: BrukerTokenInfo,
+    ) {
+        val behandling =
+            hentBehandling(behandlingId)
+                ?: throw InternfeilException("Kunne ikke oppdatere videreført opphør fordi behandlingen ikke finnes")
```

Comment:
```suggestion
                ?: throw NotFoundException("FANT_IKKE_BEHANDLING", "Kunne ikke oppdatere videreført opphør fordi behandlingen ikke finnes")
```

siden det er noe galt med forespørselen og det ikke bør være noe som prodlogges (eller kanskje det burde det?)